jars/src/org/toast/activegates/Startup.kt

```diff
@@ -0,0 +1,76 @@
+package org.toast.activegates
+
+import com.fs.starfarer.api.BaseModPlugin
+import com.fs.starfarer.api.Global
+import com.fs.starfarer.api.impl.campaign.ids.Tags
+
+class Startup : BaseModPlugin() {
+
+    /** Vanilla tag for a gate.
+     * We only want to touch gates that aren't blacklisted, so throughout this mod,
+     * only get references to gates via TAG_GATE_CANDIDATE.
+     **/
+    private val TAG_GATE = Tags.GATE
+
+    override fun onGameLoad(newGame: Boolean) {
+        super.onGameLoad(newGame)
+
+        tagPossibleGateDestinations()
+    }
+
+    private fun tagPossibleGateDestinations() {
+        val blacklistedSystems = try {
+            val jsonArray = Global.getSettings()
+                    .getMergedSpreadsheetDataForMod("id", "data/active-gates/active-gates_system_blacklist.csv", Strings.modName)
```

Comment:
since i know nothing, does this mean that we search for the file "data/active-gates/active-gates_system_blacklist.csv" in the Starsector mod directory?